math/counter.py

```diff
@@ -0,0 +1,62 @@
+from heapq import nlargest
+from operator import itemgetter
+
+
+class Counter(dict):
+    """
+    Counter merupakan Class untuk menghitung data-data unique
+    pada setiap nilai.
+
+    Args:
+        dict: parameter bersifat map.
+
+    """
+
+    def __init__(self, iterables=None, **kwargs):
```

Comment:
```suggestion
    def __init__(self, iterable: Optional[Dict[Any, int]] = None, **kwargs: int) -> None:
```
formatting kodenya agar tidak tercatch dengan ``flake8``